x-pack/plugins/cloud_security_posture/common/types/rules/v3.ts

```diff
@@ -157,25 +157,33 @@ export type CspBenchmarkRulesBulkActionRequestSchema = TypeOf<
   typeof cspBenchmarkRulesBulkActionRequestSchema
```

Comment:
the latest version now is `v4` introduced in https://github.com/elastic/kibana/pull/173345 . Though I'm not sure if we need a v4 tbh. If we don't really need v4, let's consider reverting to having v3